### PR TITLE
Add Windows ltsc2022

### PR DESCRIPTION
**Pull request by oxr463**
We might need to adjust the version name.

### CODE REVIEW

**Review by luthermonson — APPROVED** [review]

**Comment by luthermonson** [comment]
gotta figure out why the server isnt picking up the run but the yaml is good

**Comment by rosskirkpat** [comment]
need to rebase based on latest changes as 2004 has been removed. Apologies!

File: manifest.tmpl [hunk 1]
```diff
@@ -37,4 +37,11 @@ manifests:
...
-      version: 20H2
\ No newline at end of file
+      version: 20H2
+      
+  -
+    image: rancher/dapper:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}-windows-amd64-ltsc2022
+    platform:
+      architecture: amd64
+      os: windows
+      version: ltsc2022
```
Comment (phillipsj) [inline manifest.tmpl:47]: This needs to be 2022.

**Review by phillipsj — DISMISSED** [review]

File: manifest.tmpl (hunk 1 above)
Comment (luthermonson) [reply manifest.tmpl:47]: le sigh

**Comment by luthermonson** [comment]
sorry @oxr463 you on vacay and i gotta get this in, closing in favor of my pr https://github.com/rancher/dapper/pull/104

**Review by phillipsj — COMMENTED** [review]
LGTM

File: manifest.tmpl (hunk 1 above)
Comment (luthermonson) [reply manifest.tmpl:47]: turns out this is ok

**Review by luthermonson — APPROVED** [review]

**Pull request merged**